client/src/api/applicationAndTaskSelection.js

```diff
@@ -8,22 +8,15 @@
  * @param {String} application - String with currently selected application from ApplicationDropdown
  */
```

Comment:
I would rename this file to functionalities as I think it would be better fitting with the new API schema.
I general I would restructure the api folder to have one file functionalities, one file robots and one file users which would contain all the respective api calls. I think that it would be easier to work with and also easier to extend with new api calls.